designs/virtualization/4.x/create-vm-dropdowns/create-vm-dropdowns.md

```diff
@@ -0,0 +1,121 @@
+---
+parent: Virtualization
+version: 4.x
+---
+
+# create VM dropdowns
+
+This item will cover various updates to the VM creation wizard
+- Adding descriptions to the dropdown fields (using this pattern https://github.com/patternfly/patternfly-design-kit/issues/218)
+- Renaming fields
+- Reordering fields
+- Adding 'Operating System source' step
+
+
+![Create VM - General-default](img/1-0.jpg)
```

Comment:
does it make sense to have the common template selectable? IMO, it could be enough to just show the resolved common template in the review

Reply:
Based on our discussion in a previous comment thread, I thought we needed to provide a way to directly select common templates for cases there will be more than one option available (edge case, but still needed to be solved).

If this is not the case, I'll be happy to:
a. Just hide the common templates altogether
b. Just to mention what is the selected common template based on the previous selections

WDYT?

Reply:
Where does exposing the Common template come from? AFAIK we don't want to get into these details. It's more confusing IMO. If we are showing it at all we need to teach the user what they are - they won't know anything about them at this point right?

Reply:
Adding @jelkosz comment:
> Common template is not something I would tell the user about
we should just pick the most recent one (in case there are more than one common templates available after selecting all the fields above)
it is the concept which keeps confusing everyone who learns about the fact they exist

Removing the Common templates from the UI

Reply:
The user doesn't have to care about them if they see one line mentioning this in the review.

If you want our users to think our wizard runs on magic, then sure hide them.